### ISSUE TITLE
Float restricted to 1.0

### ISSUE THREAD

**Issue by cchance27** [numz/Comfyui-FlowChain#9]
**Describe the bug**
The float node and it's input on the workflow node, both seem to be restricted to max 1.00 this makes it unusable for things like CFG and Guidance. Please set the max much higher or remove the max perhaps.

**To Reproduce**
1. Create Workflow Input for FLOAT
2. Try to use 3.5
3. Gets set to 1.0

**Expected behavior**
Floats to accept all float values

**Screenshots**
None

**Desktop (please complete the following information):**
 - OS: OSX
 - Browser Arc

**Additional context**
Pretty self explanatory seems to be from 
```            
['',{default: input_value,callback: (val) => console.log('VALUE', val), "min": 0.00, "max": 1.00, "step": 0.01},],
```

**Comment by numz** [comment]
Yes true, i'll add a min max value in définition node in next update for float/int

**Comment by numz** [comment]
Hi, 
Sorry about that and the delay, I'have updated the extension, update and restart from scratch, old config will no longer works.
Now you must use standart format, (I have remove the dependency to API format) so just use a standart workflow with workflowInput or workfowOutput nodes.
Let me know
I close this issues, open a new one for the new version if needed
Regards
